UserScrape/crawler.py

```diff
@@ -109,39 +111,26 @@ def login(self) -> CrawlResult:
         passwordUrl = wd.current_url
         wd.find_element_by_css_selector('#passwordNext').click()
 
-        time.sleep(1)
         self.__log_info('password_entered')
 
-        url = urlparse(wd.current_url)
-        if url.netloc != "www.youtube.com":
-            verify = wd.find_element_by_css_selector('#authzenNext')
-            if(verify):
-                verify.click()
-
-                # verify, at least on my account presents a number to enter on the phone
-                figure: WebElement = WebDriverWait(wd, 2).until(
-                    EC.text_to_be_present_in_element(
-                        (By.CSS_SELECTOR, 'figure > samp'))
-                )
-
-                self.sendMessageToUser(f'Enter {figure.text} on your phone')
-
-                # wait for 5 minutes for an IRL meat-person to verify
-                WebDriverWait(wd, 5*60).until(EC.url_changes(wd.current_url))
-                newUrl = urlparse(wd.current_url)
-                if newUrl.netloc != "youtube.com":
-                    return CrawlResult(True, f'did not navigate to youtube after verifying (url:{wd.current_url})')
-
-            return CrawlResult(True, f'did not nvagate to youtube after password (url:{wd.current_url})')
```

Comment:
I removed all of this now, but you can explain to me how you wanted to use the CrawlResult

Reply:
Happy to remove CrawlResult and throw exceptions instead. Sometimes it's better to have a result object with success/failure but I'm not sure about what is good python.